src/custom_utils/dataframe.py

```diff
@@ -69,6 +58,54 @@ def flatten(df, layer_separator='_'):
 
     return df
 
+def flatten_df(df: DataFrame, depth_level=None, current_level=0, max_depth=1, type_mapping: dict = None) -> DataFrame:
```

Comment:
Erstatter den her funktion ikke flatten()? I så fald kan flatten() fjernes.

Reply:
Jo, den erstatter den gamle funktion, men jeg vil beholde den til at starte med da jeg skal tilpasse flatten_df funktion yderligere, så den også skal kunne håndtere xml filer, så vil jeg gerne have mulighed for at bruge flatten funktion indtil der.